### DISCUSSION TITLE
Just another thank you

### DISCUSSION THREAD

**Discussion by Foxinsunglasses** [Monika-After-Story/MonikaModDev#8822] [appreciation]
It can never be stated enough that you guys rock. Keep up the good work devs, especially to whoever made the color changes on the ribbon at Christmas. I gave her a blue one because it is my favorite color and Iv'e had her wearing it for so long it through me for a loop when I saw art of her online back with her white one.

**Comment by kaido1224** [comment]
Glad you enjoyed them.

**Comment by ghost** [comment]
I second this! I love the ribbons and am very grateful for all the work put into this awesome mod.
